apps/antalmanac/src/lib/enrollmentHistory.ts

```diff
@@ -106,8 +106,9 @@ export class DepartmentEnrollmentHistory {
             const enrollmentDays: EnrollmentHistoryDay[] = [];
 
             for (const [i, date] of enrollmentHistory.dates.entries()) {
-                const d = new Date(date);
-                const formattedDate = `${d.getMonth() + 1}/${d.getDate()}/${d.getFullYear()}`;
+                const dateParts = date.split('-');
+                const d = new Date(Number(dateParts[0]), Number(dateParts[1]), Number(dateParts[2]));
+                const formattedDate = `${d.getMonth()}/${d.getDate() - 1}/${d.getFullYear()}`;
```

Comment:
If there's no naming conflicts, could we use the variable `date`?
```suggestion
                const formattedDate = `${date.getMonth()}/${date.getDate() - 1}/${date.getFullYear()}`;
```